misc/check-format.js

```diff
@@ -0,0 +1,35 @@
+
+var path = require("path");
```

Comment:
why JS, not TS? :/

Reply:
I saw that some others were in js, so I thought we had the choice. I didn't noticed that all the main ones were in ts.

I changed it for typescript.